proto/ecosystem-incentive/genesis.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+package ununifi.ecosystemincentive;
+
+import "gogoproto/gogo.proto";
+import "ecosystem-incentive/params.proto";
+
+option go_package = "github.com/UnUniFi/chain/x/ecosystem-incentive/types";
+
+// GenesisState defines the ecosystemincentive module's genesis state.
+message GenesisState {
+  Params params = 1 [(gogoproto.nullable) = false];
```

Comment:
Does it include all the state exports? I think SubjectInfo should be included here.

Reply:
for what?
for restart of the network or something like this case?

Reply:
Yes, also for hardfork case.

Reply:
Got it.
I modified to achieve it.
please check it.